### ISSUE TITLE
Create pages Get Involved and Get In Touch

### ISSUE THREAD

**Issue by domingoruiz** [rocky-linux/rockylinux.org#10]
Hi. I have the idea that cretate a Get Involved page and Get In Touch page. If you agree with the idea, I could take care of it and propose a pull request. If you agree, it would be good if you assigned me the issue. I am also all ears for proposals for those pages.

Thanks you.

Bye.

**Comment by FoggyMtnDrifter** [comment]
I think we should hold off on any further changes for now. @hbjydev is working on a custom component library for us to use on a future version of the site, as currently TailwindUI's licensing would prevent users who don't own a license from contributing.

**Comment by FoggyMtnDrifter** [comment]
I do, however, think this would be a great idea for the future site!

**Comment by hbjydev** [comment]
@ressonix That's gonna take me a long while to build out yet, so don't stop making improvements to this on my count just yet. Not unless they're Tailwind UI stuff - just try not to implement any more of that.

**Comment by FoggyMtnDrifter** [comment]
Okay, so we should try not to use any more TailwindUI components on the site and start implementing custom components? Just wanted to get further clarification on that part.

**Comment by FoggyMtnDrifter** [comment]
While I agree that Bootstrap would be easier to contribute with, I feel that TailwindCSS offers more flexibility and the ability to build custom components quickly. TailwindCSS also has tons of plugin support and makes it super easy to optimize our CSS files from the get-go.

It's one of those situations where one can be beneficial but limiting, and the other can be beneficial and more flexible but more complicated to set up.

**Comment by domingoruiz** [comment]
Another option is the combination of Hugo and Bootstrap. It would be much simpler than the current one and we would continue to have the advantages.

**Comment by shadonna-bossy** [comment]
> Another option is the combination of Hugo and Bootstrap. It would be much simpler than the current one and we would continue to have the advantages.

[@sha256 is the owner and I'm not a developer at all neither or my children but I've gotten lost in this mess trying to work from home and I'm a student. Somehow my account is all over the place and someone is profiting from my accounts that I'm trying to find. If you can help me I need help but I'm not trying to pay anymore money I've been swindled enough. My next step is to go to the FTC since this has been going on for a year now and someone has access to all my bank information and school accounts ]()

**Comment by domingoruiz** [comment]
@hbjydev Can you link the issue to me to get to it? 

Thank you.

**Comment by hbjydev** [comment]
I don't think this page is something we really need. Let's take this to Slack and discuss there for now.

**Comment by hbjydev** [comment]
We need to get the content/documentation team involved for pages like that.
